jwst/associations/tests/test_standards.py

```diff
@@ -0,0 +1,139 @@
+"""Test against Level2 standard associations
```

Comment:
This test fails for me on your new standards file. The truth has 4 associations, while the test generated 8.

Reply:
James wants me to wait until after the merge to update the truth files.